Docs/source/install/hpc/lassen.rst

```diff
@@ -37,6 +37,34 @@ We recommend to store the above lines in a file, such as ``$HOME/lassen_warpx.pr
 
    source $HOME/lassen_warpx.profile
 
+And since Lassen does not yet provide a module for them, install ADIOS2, BLAS++ and LAPACK++:
+
+.. code-block:: bash
+
+   # c-blosc (I/O compression)
+   git clone -b v1.21.1 https://github.com/Blosc/c-blosc.git src/c-blosc
+   rm -rf src/c-blosc-lassen-build
+   cmake -S src/c-blosc -B src/c-blosc-lassen-build -DBUILD_TESTS=OFF -DBUILD_BENCHMARKS=OFF -DDEACTIVATE_AVX2=OFF -DCMAKE_INSTALL_PREFIX=$HOME/sw/lassen/c-blosc-1.21.1
+   cmake --build src/c-blosc-lassen-build --target install --parallel 16
+
+   # ADIOS2
+   git clone -b v2.7.1 https://github.com/ornladios/ADIOS2.git src/adios2
+   rm -rf src/adios2-lassen-build
+   cmake -S src/adios2 -B src/adios2-lassen-build -DADIOS2_USE_Blosc=ON -DADIOS2_USE_Fortran=OFF -DADIOS2_USE_Python=OFF -DADIOS2_USE_ZeroMQ=OFF -DCMAKE_INSTALL_PREFIX=$HOME/sw/lassen/adios2-2.7.1
```

Comment:
No need for SST yet and the found `/usr/lib64/libfabric.so` will most likely not be the one we need (once we need it, we can check which one is fine):
```suggestion
   cmake -S src/adios2 -B src/adios2-lassen-build -DADIOS2_USE_Blosc=ON -DADIOS2_USE_Fortran=OFF -DADIOS2_USE_Python=OFF -DADIOS2_USE_SST=OFF -DADIOS2_USE_ZeroMQ=OFF -DCMAKE_INSTALL_PREFIX=$HOME/sw/lassen/adios2-2.7.1
```